### PR TITLE
MES-6806 CPC End to end page flow

### PR DESCRIPTION
**Pull request by Cullima**
## Description
[MES-6806](https://jira.dvsacloud.uk/browse/MES-6806)
- End to end test flow (skeleton pages) 

## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/pages/back-to-office/cat-cpc/back-to-office.cat-cpc.page.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  ngOnInit() {
+  }
+
+  navigateBack(): void {
+    this.navController.back();
+  }
+
+  async navigateForward(): Promise<void> {
+    await this.routeByCat.navigateToPage(TestFlowPageNames.OFFICE_PAGE, TestCategory.B);
```
Comment (andrewsetterfield) [inline src/app/pages/back-to-office/cat-cpc/back-to-office.cat-cpc.page.ts]: wrong category

**Review by andrewsetterfield — CHANGES_REQUESTED** [review]

File: src/app/pages/back-to-office/cat-cpc/back-to-office.cat-cpc.page.ts (hunk 1 above)
Comment (Cullima) [reply src/app/pages/back-to-office/cat-cpc/back-to-office.cat-cpc.page.ts]: argh. thanks!

Comment (tomcrawleyy) [inline src/app/pages/back-to-office/cat-cpc/back-to-office.cat-cpc.page.ts]: wrong TestCategory

Comment (Cullima) [reply src/app/pages/back-to-office/cat-cpc/back-to-office.cat-cpc.page.ts]: fixed

**Review by andrewsetterfield — APPROVED** [review]

**Review by tomcrawleyy — APPROVED** [review]

**Pull request merged**